hassio/src/addon-store/hassio-addon-store.ts

```diff
@@ -206,6 +206,7 @@ export class HassioAddonStore extends LitElement {
     fireEvent(this, "supervisor-collection-refresh", {
       collection: "supervisor",
     });
+    fireEvent(this, "supervisor-collection-refresh", { collection: "store" });
```

Comment:
Do we need this here if the dialog already does a refresh after adding a store?

Reply:
You're right, we don't – I only added it for consistency. Removed.